x509-ocsp/src/ext.rs

```diff
@@ -138,9 +145,8 @@ impl AssociatedOid for AcceptableResponses {
 /// ```
 ///
 /// [RFC 6960 Section 4.4.4]: https://datatracker.ietf.org/doc/html/rfc6960#section-4.4.4
-pub struct ArchiveCutoff(GeneralizedTime);
+pub type ArchiveCutoff = OcspGeneralizedTime;
```

Comment:
Since this is a type alias, doesn't in mean the `OcspGeneralizedTime` now has a `AssociatedOid`?
```suggestion
pub struct ArchiveCutoff(pub OcspGeneralizedTime);
```

Reply:
That makes sense. We don't really want the ArchiveCutoff OID to be associated to `OcspGeneralizedTime` itself.